components/DropdownLight/DropdownLight.jsx

```diff
@@ -142,10 +157,65 @@ const SelectionListItem = styled.li`
   }}
 `;
 
-const DropdownLight = ({ disabled, items, theme, placeholder, name }) => {
+const InputWrapper = styled.div`
+  position: relative;
+`;
+
+const InputText = styled(TextInput)`
+  ${({
+    theme: {
+      spacing: { xsmall },
+    },
+  }) => css`
+    margin-top: ${xsmall}px;
+  `}
+`;
+
+const InputIcon = styled(Icon)`
+  cursor: pointer;
+  position: absolute;
+  ${({
+    theme: {
+      spacing: { xsmall, medium },
+      baseFontSize,
+    },
+  }) => css`
+    right: ${medium}px;
+    bottom: ${xsmall * 1.25}px;
+    width: ${baseFontSize * 1.5}px;
+  `}
+`;
+
+const InputErrorIcon = styled(InputIcon).attrs({ name: 'error' })`
+  ${({
+    theme: {
+      colors: {
+        error: { 700: error700 },
+      },
+    },
+    skin,
+  }) => css`
+    color: ${skin === 'default' ? error700 : 'inherit'};
+  `}
+`;
```

Comment:
How about removing the InputIcon and doing all the styling for it in InputErrorIcon?

Reply:
I evaluated better and decided to remove o inputerror